app/src/main/res/layout/fragment_get_involved.xml

```diff
@@ -54,6 +54,6 @@
     android:layout_height="wrap_content"
     android:layout_alignParentRight="true"
     android:layout_alignParentTop="true"
-    android:elevation="4dp"
```

Comment:
This fixes the "Fork me" badge in this page. Elevation in material design puts the card over this Image. Just a random 8dp elevation fixed the problem. I don't know what else we can do here if we don't like this. 
In devices below Lollipop, it is fine.